frontend/components/meta.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import Head from 'next/head';
+
+const description = `We serve fast and scalable informational images as badges
+for GitHub, Travis CI, Jenkins, WordPress and many more services. Use them to
```

Comment:
This is going to insert literal newlines in your text? Is that what you want? You might want to use string concatenation instead.